dev/com.ibm.ws.smallrye.reactive.streams.operators.1.0/bnd.bnd

```diff
@@ -0,0 +1,40 @@
+#*******************************************************************************
+# Copyright (c) 2019 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#    IBM Corporation 
+# This wraps the external JAR.
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+javac.source: 1.8
+javac.target: 1.8
+
+Bundle-SymbolicName: com.ibm.ws.smallrye.reactive.streams.operators.1.0; singleton:=true
```

Comment:
We should change this to `com.ibm.ws.io.smallrye.reactive.streams.operators.1.0`

Reply:
Edit done.